src/lib/SCEditor.js

```diff
@@ -404,16 +404,30 @@ define(function (require) {
 
 		/**
 		 * Creates the editor iframe and textarea
+		 * @param {boolean} startInSourceMode Force loading the editor in this
+		 *																	mode
 		 * @private
 		 */
-		initEditor = function () {
+		initEditor = function (startInSourceMode) {
 			var doc, tabIndex;
 
-			$sourceEditor  = $('<textarea></textarea>').hide();
+			$sourceEditor  = $('<textarea></textarea>');
 			$wysiwygEditor = $(
 				'<iframe frameborder="0" allowfullscreen="true"></iframe>'
 			);
 
+			/* This needs to be done right after they are created because,
```

Comment:
There's a [check for](https://github.com/samclarke/SCEditor/blob/master/src/lib/SCEditor.js#L344) `isWysiwygSupported` in the `init()` method. Changing that to include the `startInSourceMode` option should save a few bytes as it's just:

    if (!browser.isWysiwygSupported || options.startInSourceMode) {